test/e2e/operator/deployment_api.go

```diff
@@ -52,7 +52,7 @@ func getDeployment(name string) api.Deployment {
 var _ = deploy.DescribeForSome("API", func(d *deploy.Deployment) bool {
 	// Run these tests for all bare operator deployments, i.e.
 	// those which did not already install the driver.
-	return d.HasOperator && !d.HasDriver
+	return (d.HasOLM || d.HasOperator) && !d.HasDriver
```

Comment:
Isn't `d.HasOperator` also set for "olm-operator"? A deployment that just has OLM but not the operator shouldn't be used for the API tests because the operator won't be running...

IMHO the entire line has to stay as it was.

Reply:
Makes sense. I reverted this change.